src/api/app/views/webui2/webui/interconnects/_interconnect_button.html.haml

```diff
@@ -1,23 +1,22 @@
 .list-group-item{ data: { interconnect: name } }
   .d-flex.w-100.justify-content-between
-    %h5.mb-1{ class: "#{'text-muted' unless exists}" }= title
-    %small.text-primary.connected{ class: "#{'d-none' unless exists}" } Connected
+    %h5{ class: "#{'text-muted' unless exists}" }= title
+    .text-primary.connected{ class: "#{'d-none' unless exists}" } Connected
     - unless exists
       - params = "project[name]=#{u(name)}&project[remoteurl]=#{remoteurl}&project[title]=#{title}&project[description]=#{description}"
       = button_tag data: { url: interconnects_path,
                            params: params,
                            remote: true,
                            disable_with: 'Connecting...',
                            method: :post },
-                   class: 'btn btn-sm btn-primary interconnect' do
+                   class: 'btn btn-primary interconnect' do
         Connect
-  .interconnect-info.text-muted
-    %i.fa.fa-cubes.fa-xs{ class: "#{'text-secondary' if exists}" }
+  %div{ class: "#{'text-muted' unless exists}" }
+    %i.fa.fa-cubes{ class: "#{'text-secondary' if exists}" }
     - if exists
```

Comment:
I suggest replacing the `if/else` by `link_to_if`:

`= link_to_if(exists, name, project_show_path(project: name))`

Reply:
Nice! Added.